lib/joken/signer.ex

```diff
@@ -90,6 +97,8 @@ defmodule Joken.Signer do
     )
   end
 
+  def create(alg, key, headers) when alg in @map_key_algorithms, do: raise(Joken.Error, :algorithm_needs_key)
```

Comment:
This will generate compiler warnings. Please, prefix the unused variables with underscore.

```suggestion
  def create(alg, _key, _headers) when alg in @map_key_algorithms, do: raise(Joken.Error, :algorithm_needs_key)
```